.mage/js.go

```diff
@@ -229,8 +250,13 @@ func (js Js) ServeMain() error {
 
 // Messages extracts the frontend messages via babel.
 func (js Js) Messages() error {
+	isCI := os.Getenv("CI") == "true"
+	cacheExists := pathExists("./.cache/messages")
```

Comment:
This should be handled via `target` only, see https://github.com/TheThingsNetwork/lorawan-stack/pull/2751

Reply:
See https://github.com/TheThingsNetwork/lorawan-stack/pull/2764#discussion_r442275560